packages/tldraw/src/lib/shapes/line/toolStates/Pointing.ts

```diff
@@ -45,8 +46,8 @@ export class Pointing extends StateNode {
 				this.editor.getShapeParentTransform(this.shape)!,
 				new Vec(this.shape.x, this.shape.y)
 			)
-
-			const nextPoint = Vec.Sub(currentPagePoint, shapePagePoint).addXY(0.1, 0.1)
+			const nudgedPoint = Vec.Sub(currentPagePoint, shapePagePoint).addXY(0.1, 0.1)
```

Comment:
i know this isn't your code but a comment here about why it's being nudged would be helpful — do you know why?

Reply:
It's to avoid zero-length lines, I can add a comment for sure.